custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -888,6 +888,7 @@ def _prepare_docker_command(self, options, run_mode, raw_arguments):
         in_docker_fit_output_dir = "/opt/fit_output_dir"
         in_docker_fit_target_filename = "/opt/fit_target.csv"
         in_docker_fit_row_weights_filename = "/opt/fit_row_weights.csv"
+        in_docker_runtime_parameters_file = "/opt/runtime_parameters.yaml"
```

Comment:
what does all of the `in_docker_` stuff mean?  do you mean that this is the where the runtime parameters file is mounted in the docker container?  if so, maybe these should change to `file_name_docker_mounted_location` or ... something?  asking chat gpt gives me:

- docker_mounted_params_file="/opt/params.yaml"
- docker_params_file_location="/opt/params.yaml"
- docker_params_file_mount_point="/opt/params.yaml"
- docker_params_file_path="/opt/params.yaml"
- docker_params_file_destination="/opt/params.yaml"

maybe like these but move `docker`

- params_file_docker_location
- params_file_location_in_docker

:shrug: not sure what the best answer is, but `in_docker_` was certainly confusing for a first-time reader.

Reply:
I will have to do this renaming of all vars in separate PR, including a set of Unit test for each particular case

Reply:
yeah.  this is definitely out of scope